device-types/Palo Alto/PA-460.yml

```diff
@@ -0,0 +1,44 @@
+---
+manufacturer: Palo Alto
+model: PA-460
+slug: pa-460
+part_number: PA-460
+u_height: 1
+is_full_depth: false
+comments: '[Product Summary Specsheet - PAN-OS 10.1](https://www.paloaltonetworks.com/resources/datasheets/palo-alto-networks-product-summary-specsheet-pan-os-10-1)'
+power-ports:
+  - name: PS1
```

Comment:
Please correct me if I am wrong, but it appears that the power ports on these are `dc-terminals` not `nema-1-15p` ports